readthedocs/config/config.py

```diff
@@ -135,25 +135,40 @@ class BuildConfigBase(object):
     version = None
 
     def __init__(self, env_config, raw_config, source_file, source_position):
```

Comment:
I'll open an issue about removing source_position, I think we have a card to track this.

Reply:
https://github.com/rtfd/readthedocs.org/issues/4705